android/src/main/java/com/lwansbrough/RCTCamera/RCTCameraModule.java

```diff
@@ -751,6 +760,29 @@ public void onPictureTaken(byte[] data, Camera camera) {
         }
     }
 
+    private void writeLocationExifData(File cameraRollFile, ReadableMap options) {
+        if(!options.hasKey("metadata"))
+            return;
+
+        ReadableMap metadata = options.getMap("metadata");
+        if (!metadata.hasKey("location"))
+            return;
+
+        ReadableMap location = options.getMap("location");
+        if(!location.hasKey("coords"))
```

Comment:
AFAICT this means the metadata option will need both location and coords in order to be able to retrieve either? Is this on purpose?

assuming I understand this correctly, imo there are two reasonable options here
- change the option from `metadata: object` to something like `exifGeo: boolean`
- console.error in JS if `location XOR coords`

Thoughts?